lib/naughty/core.lua

```diff
@@ -420,159 +318,178 @@ function naughty.getById(id)
     end
 end
 
---- Increase notification ID by one
-function naughty.get_next_notification_id()
-  counter = counter + 1
-  return counter
-end
-
---- Install expiration timer for notification object.
--- @tparam notification notification Notification object.
--- @tparam number timeout Time in seconds to be set as expiration timeout.
-local function set_timeout(notification, timeout)
-    local die = function (reason)
-        naughty.destroy(notification, reason)
-    end
-    if timeout > 0 then
-        local timer_die = timer { timeout = timeout }
-        timer_die:connect_signal("timeout", function() die(naughty.notificationClosedReason.expired) end)
-        if not suspended then
-            timer_die:start()
-        end
-        notification.timer = timer_die
-    end
-    notification.die = die
-end
-
 --- Set new notification timeout.
+--
+-- This function is deprecated, use `notification:reset_timeout(new_timeout)`.
+--
 -- @tparam notification notification Notification object, which timer is to be reset.
 -- @tparam number new_timeout Time in seconds after which notification disappears.
--- @return None.
+-- @deprecated naughty.reset_timeout
 function naughty.reset_timeout(notification, new_timeout)
-    if notification.timer then notification.timer:stop() end
+    gdebug.deprecate("Use notification:reset_timeout(new_timeout)", {deprecated_in=5})
+    notification:reset_timeout(new_timeout)
+end
 
-    local timeout = new_timeout or notification.timeout
-    set_timeout(notification, timeout)
-    notification.timeout = timeout
+--- Replace title and text of an existing notification.
+--
+-- This function is deprecated, use `notification.text = new_text` and
```

Comment:
I see above that `message` is replacing `text`. Is this `text` still used here?

Reply:
done, thanks!